automation_testing/tests/Components/Checkbox/checkbox.spec.ts

```diff
@@ -31,4 +31,12 @@ test.describe.parallel("Components - Checkbox - Kitchen Sink", () => {
 			await checkboxPage.validateFontColorFromElement(checkboxPage.checkboxIcon.nth(i), expectBgColor, false);
 		}
 	});
+
+	test("Validate Checkbox has almostBlack color as text.", async () => {
```

Comment:
Checkboxes don't use almostBlack for the text, they use grey4. Skipping this test for now until we update the references for grey4